walrus:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  front_left_pod_joint_controller:
    type: effort_controllers/JointEffortController
    joint: walrus/front_left_pod_joint
  back_left_pod_joint_controller:
    type: effort_controllers/JointEffortController
    joint: walrus/back_left_pod_joint
  front_right_pod_joint_controller:
    type: effort_controllers/JointEffortController
    joint: walrus/front_right_pod_joint
  back_right_pod_joint_controller:
    type: effort_controllers/JointEffortController
    joint: walrus/back_right_pod_joint

  left_drive_controller:
    type: effort_controllers/JointEffortController
    joint: walrus/left_drive_joint
  right_drive_controller:
    type: effort_controllers/JointEffortController
    joint: walrus/right_drive_joint
